volmdlr/edges.py

```diff
@@ -247,6 +245,10 @@ class Line2D(Line):
     """
 
     def __init__(self, point1, point2, *, name=''):
+        if isinstance(point1, list):
```

Comment:
isinstance(obj, list) isn't robust enough to check for an object to be a sequence. If we are to be less strict on the way to provide data to init, tuples should be ok as well. dessia_common defines a way to check it. Consider using is_sequence instead

```suggestion
        if dc.is_sequence(point1):
```